lib/cdo/contact_rollups.rb

```diff
@@ -152,7 +152,6 @@ def self.build_contact_rollups
     update_roles
     update_grades_taught
     update_ages_taught
-    update_district
```

Comment:
If `update_district` is not being invoked anymore, can the method definition be removed?

Reply:
oh, actually I think I shouldn't be preventing this from being invoked, since we are still updating `school_districts`, just not `districts`